python.d/dnsdist.chart.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+from json import loads
+from bases.FrameworkServices.UrlService import UrlService
+
+ORDER = ['latency_rel', 'latency_abs', 'cache_rel', 'cache_abs', 'acl_abs', 'invalid_abs', 'queries_rel', 'queries_abs', 'health_abs']
+CHARTS = {
+	'latency_rel': {
+		'options': [None, 'Average latency', 'value', 'Latency', 'dnsdist.latency_rel', 'area'],
+		'lines': [
+			['latency-slow', '> 1sec', 'incremental'],
+			['latency100-1000', '100-1000ms', 'incremental'],
+			['latency50-100', '50-100ms', 'incremental'],
+			['latency10-50', '10-50ms', 'incremental'],
+			['latency1-10', '1-10ms', 'incremental'],
+			['latency0-1', '< 1ms', 'incremental']
+	]},
+	'latency_abs': {
+		'options': [None, 'Absolute latency', 'value', 'Latency', 'dnsdist.latency_abs', 'line'],
+		'lines': [
+			['latency-slow', '> 1sec', 'absolute'],
+			['latency100-1000', '100-1000ms', 'absolute'],
+			['latency50-100', '50-100ms', 'absolute'],
+			['latency10-50', '10-50ms', 'absolute'],
+			['latency1-10', '1-10ms', 'absolute'],
+			['latency0-1', '< 1ms', 'absolute']
+	]},
+	'cache_rel': {
+		'options': [None, 'Relative cache stats', 'value', 'Cache', 'dnsdist.cache_rel', 'area'],
+		'lines': [
+			['cache-hits', 'hits', 'incremental'],
+			['cache-misses', 'misses', 'incremental']
+	]},
+	'cache_abs': {
+		'options': [None, 'Absolute cache stats', 'value', 'Cache', 'dnsdist.cache_abs', 'area'],
```

Comment:
family `Cache` -> `cache`
chart type `area` -> `stacked`
dimension algorithm `absolute` -> `percentage-of-absolute-row`